### PR TITLE
lapack: Prepare for compilation with LLVM Flang.

### PR DESCRIPTION
**Pull request by mmuetzel**
Not sure if we should really bump the pkgrel already. The existing packages shouldn't be changing.
Please, let me know and I'll revert that part.

### CODE REVIEW

File: mingw-w64-lapack/PKGBUILD [hunk 1]
```diff
@@ -28,37 +28,42 @@ build() {
...
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
     ${MINGW_PREFIX}/bin/cmake.exe \
       -G"Ninja" \
       -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
-      -DCMAKE_Fortran_COMPILER=${MINGW_PREFIX}/bin/gfortran.exe \
       -DCMAKE_BUILD_TYPE=${_btype} \
       -DBUILD_SHARED_LIBS=ON \
       -DCMAKE_NEED_RESPONSE=ON \
-      -DBUILD_TESTING=ON \
+      -DBUILD_TESTING=OFF \
```
Comment (lazka) [inline mingw-w64-lapack/PKGBUILD:46]: just wondering, why?

Comment (mmuetzel) [reply mingw-w64-lapack/PKGBUILD:46]: It was failing to link the tests with the library.
I'll re-do that now it is [likely fixed](https://reviews.llvm.org/D145389).

Comment (lazka) [reply mingw-w64-lapack/PKGBUILD:46]: ah, ok

**Comment by mmuetzel** [comment]
Given that LAPACK crashes on CLANG64 for some functions with complex input, I'm not sure if we should package it.
Preparing the build rules might still make sense in case someone would like to build it manually and experiment with it.

**Comment by lazka** [comment]
lgtm

**Pull request merged**